photon-ml/src/main/scala/com/linkedin/photon/ml/supervised/classification/SmoothedHingeLossLinearSVMModel.scala

```diff
@@ -69,3 +70,14 @@ class SmoothedHingeLossLinearSVMModel(override val coefficients: Coefficients)
     }
   }
 }
+
+object SmoothedHingeLossLinearSVMModel {
+  /**
+   * Create a new smoothed hinge loss SVM model with the provided coefficients (means) and variances
+   *
+   * @param coefficients The feature coefficient means and variances for the model
+   * @return A smoothed hinge loss SVM model
+   */
+  def createModel(coefficients: Coefficients): SmoothedHingeLossLinearSVMModel =
+    new SmoothedHingeLossLinearSVMModel(coefficients)
```

Comment:
I noticed that this PR also introduces many factory style methods.

Just out of curiosity, what's the motivation behind such choice? Do we plan to hide or protect the implementation details of the constructors?